youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,65 @@
+from .common import InfoExtractor
+from ..utils import get_element_by_class, compat_urlparse, clean_html
+import re
+
+
+class WikimediaIE(InfoExtractor):
+    _NETRC_MACHINE = 'wikimediaorg'
+    IE_NAME = 'wikimedia.org'
+    _API_BASE_URL = 'https://commons.wikimedia.org/'
+    _VALID_URL = r'https://commons.wikimedia.org/wiki/File:(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://upload.wikimedia.org/wikipedia/commons/transcoded/d/d7/Die_Temperaturkurve_der_Erde_%28ZDF'
```

Comment:
This should be the URL being extracted from, not the URL of the extracted media link. Have you run the test as described in the Developer section of the yt-dl manual? Or am I completely confused?
```
python test/test_download.py TestDownload.test_Wikimedia
```